lux/action/temporal.py

```diff
@@ -0,0 +1,109 @@
+#  Copyright 2019-2020 The Lux Authors.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import lux
+from lux.vis.VisList import VisList
+from lux.vis.Vis import Vis
+import pandas as pd
+from lux.core.frame import LuxDataFrame
+
+
+def temporal(ldf):
+    """
+    Generates line charts for temporal fields at different granularities.
+
+    Parameters
+    ----------
+    ldf : lux.core.frame
+            LuxDataFrame with underspecified intent.
+
+    Returns
+    -------
+    recommendations : Dict[str,obj]
+            Object with a collection of visualizations that result from the Temporal action.
+    """
+    visuals = []
+    for c in ldf.columns:
+        if ldf.data_type[c] == "temporal":
+            try:
+                generated_vis = create_vis(ldf, c)
+                visuals.extend(generated_vis)
+            except (ValueError, AttributeError):
+                pass
+    recommendation = {
+        "action": "Temporal",
+        "description": "Show trends over <p class='highlight-descriptor'>time-related</p> attributes.",
```

Comment:
Want to try adding in a long description field as well? Our long descriptions include the description and also how the graphs are sorted and other additional info for the user.

Reply:
Thanks for clarifying what the long descriptions are used for. I'll add how the graphs are sorted in the long description.